Dapper/SqlMapper.cs

```diff
@@ -2221,7 +2225,13 @@ public static object SanitizeParameterValue(object value)
 
         private static IEnumerable<PropertyInfo> FilterParameters(IEnumerable<PropertyInfo> parameters, string sql)

```

Comment:
It may get more complex later...and the calling function is already fairly complex. I'd still favor leaving this separate. It'll get inlined via JIT, but actually inlining the C# will be harder to read/maintain.